_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -59,31 +67,23 @@ CSS specifications define each declaration as being either [important][] (if it
 
 ## Test Cases
 
+Test cases description abusively refer to the CSS properties of text nodes, meaning the one of their parent.
```

Comment:
I'm not sure we can actually have text here. The act-tools script might not transfer this to the WAI site. Could you either have a look to make sure it does, or move this to the background?